roles/nodejs-variables/meta/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+dependencies:
+  - role: git-cache
+    repo: git://github.com/nodejs/node.git
+    version: "{{ nodejs_version }}"
+    variable_prefix: nodejs
+    state: present
+
```

Comment:
@zachmullen Here's the dependency on the new `git-cache` role (#17) that does all the work and sets the variables you're asking about.